sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-factory.ts

```diff
@@ -1,48 +1,61 @@
-import { MSFAANumber } from "@sims/sims-db";
+import { MSFAANumber, OfferingIntensity, Student } from "@sims/sims-db";
 import {
   E2EDataSources,
+  MSFAAStates,
   createFakeMSFAANumber,
   createFakeStudent,
   createFakeUser,
   saveFakeApplication,
   saveFakeStudent,
 } from "@sims/test-utils";
-import { MSFAATestInputData } from "./msfaa-part-time-process-integration.scheduler.models";
+import { MSFAATestInputData } from "./msfaa-helper";
 import { createFakeSINValidation } from "@sims/test-utils/factories/sin-validation";
 
 /**
  * Save an MSFAA record providing all data that will be
  * part of the MSFAA file generation.
  * @param db data source helper.
  * @param msfaa test input data.
+ * @param relations
+ * - `fakeStudent` fakeStudent to be associated with the MSFAATestInputData.
+ * @param options to saveMsfaaTestInputData.
+ * - `state` if provided, specifies the MSFAA State to be associated with the provided MSFAATestInputData.
+ * - `offeringIntensity` if provided, specifies the offering intensity associated with the provided MSFAATestInputData.
  * @returns a saved MSFAA record that uses the input test
  * data to be created.
  */
 export async function saveMSFAATestInputData(
   db: E2EDataSources,
   msfaaDataInput: MSFAATestInputData,
+  relations?: { fakeStudent?: Student },
+  options?: { state?: MSFAAStates; offeringIntensity?: OfferingIntensity },
```

Comment:
offeringIntensity is a part of `MSFAATestInputData` itself.  IMO we do not need an argument seperately to  pass.